src/ripple/nodestore/impl/DecodedBlob.cpp

```diff
@@ -31,8 +31,8 @@ DecodedBlob::DecodedBlob (void const* key, void const* value, int valueBytes)
 
         Bytes
 
-        0...3       LedgerIndex     32-bit big endian integer
-        4...7       Unused?         An unused copy of the LedgerIndex
+        0...3       Unused
```

Comment:
Same question as noted in the nodestore README.md.

Reply:
0...7 Unused